README.md

```diff
@@ -416,20 +416,21 @@
 
 ### Modules
 
-* <a name="consistent-module-structure"></a>
-  Use a consistent structure in module definitions.
-  <sup>[[link](#consistent-module-structure)]</sup>
+* <a name="use-import-alias-require"></a>
```

Comment:
@lexmag agreed, but "consistent-module-structure" wasn't a good fit IMO, it's a very general title. Alternatives?